api/resource/resource.go

```diff
@@ -157,7 +157,7 @@ func (r *Resource) DeepCopy() *Resource {
 // the resource.
 // TODO: move to RNode, use GetMeta to improve performance.
 // TODO: make a version of mergeStringMaps that is build-annotation aware
-//   to avoid repeatedly setting refby and genargs annotations
+// to avoid repeatedly setting refby and genargs annotations
```

Comment:
_No good deed goes unpunished._
```suggestion
// to avoid repeatedly setting refby and genargs annotations.
```
It's strange to see punctuation but no capitalization on these comments.